lightly/api/api_workflow_client.py

```diff
@@ -120,9 +117,9 @@ def __init__(
     def check_version_compatibility(self):
         minimum_version = get_minimum_compatible_version()
         if version_compare(__version__, minimum_version) < 0:
-            raise ValueError(f"Incompatible Version of lightly pip package. "
-                             f"Please upgrade to at least version {minimum_version} "
-                             f"to be able to access the api and webapp")
+            raise ValueError(f"Incompatible version of lightly pip package. "
+                             f"Please upgrade to at least version '{minimum_version}' "
+                             f"to be able to access the api.")
```

Comment:
Minor: What do you think about changing this function to return bool and naming is something like `is_version_compatible`?

Reply:
Actually, for this PR the relevant function is `do_version_check` which is raising, but could return a bool instead.

Up to you if you want to change or keep.

Reply:
Yes, sounds good to me 👍